packages/core/src/interpreter.ts

```diff
@@ -1053,10 +1092,20 @@ export class Interpreter<
             // Send "error.platform.id" to this (parent).
             this.send(toSCXMLEvent(errorEvent as any, { origin: id }));
           } catch (error) {
-            reportUnhandledExceptionOnInvocation(errorData, error, id);
+            if (this.parent) {
+              this.parent.send({
+                type: 'xstate.error',
```

Comment:
@davidkpiano in this case, is it ok this type of error or should it be a different one?